src/sentry/plugins/sentry_mail/activity/release.py

```diff
@@ -111,19 +111,28 @@ def get_participants(self):
             ).distinct()
         )
 
-        # get all the involved users' settings for deploy-emails
-        options_by_user_id = {
-            uoption.user_id: uoption.value
-            for uoption in UserOption.objects.filter(
-                user__in=users,
-                organization=self.organization,
-                key='deploy-emails',
-            )
-        }
+        # get all the involved users' settings for deploy-emails (user default
+        # saved without org set)
+        user_options = UserOption.objects.filter(
+            user__in=users,
+            key='deploy-emails',
+        ).filter(
+            Q(organization=self.organization) | Q(organization=None),
```

Comment:
Nitpick: You can include this in the first `filter` call as the first arg as well